__tests__/composer/install.test.ts

```diff
@@ -14,12 +14,7 @@ describe('composer.install with mocked exec', () => {
     await composer.install('locked')
 
     expect(execMock).toHaveBeenCalledTimes(1)
-    expect(execMock).toHaveBeenCalledWith('composer', [
-      'install',
-      '--no-interaction',
-      '--no-progress',
-      '--ansi'
-    ])
+    expect(execMock).toHaveBeenCalledWith('composer install --no-interaction --no-progress --ansi')
```

Comment:
This update is due to some changes on `install()` from `src/composer/install.ts`, when it parse and stringify back composer options. 

In fact it was not necessary and was source of errors when using `--working-dir subdirectory`, it would call `exec('composer', ['install', '--no-interaction', '--no-progress', '--ansi', '--working-dir', 'subdirectory'])` and this is not something we want.